src/xdocs/checks/coding/emptystatement.xml

```diff
@@ -30,14 +30,16 @@
             Example:
         </p>
         <source>
-public void foo() {
-  int i = 5;
-  if (i &gt; 3); // violation, &quot;;&quot; right after if statement
+public class Example1 {
+  public void foo() {
+    int i = 5;
+    if(i &gt; 3); // violation
     i++;
-  for (i = 0; i &lt; 5; i++); // violation
+    for (i = 0; i &lt; 5; i++); // violation
     i++;
-  while (i &gt; 10) // OK
+    while (i &gt; 10) // OK
     i++;
```

Comment:
Please remove ok trailing comments, we can keep them only it has some extra text

Reply:
do we need to remove OK trailing comments from Example1.java as well?

Reply:
Yes